packages/nextjs/components/scaffold-eth/Balance.tsx

```diff
@@ -44,6 +55,8 @@ export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
     );
   }
 
+  const formattedBalance = Number(balance?.formatted);
```

Comment:
If in case if the balance is `undefined` then `Number(undefined) => Nan` and this break all the 

And this will break further all multiplications with price, maybe its better we do : 

```ts
const formattedBalance = balance ? Number(balance.formatted) : 0; 
```

Reply:
don't copy my comment! 🤣